src/main/java/com/volasoftware/tinder/services/contracts/RatingService.java

```diff
@@ -0,0 +1,10 @@
+package com.volasoftware.tinder.services.contracts;
+
+import com.volasoftware.tinder.dtos.RatingDto;
+import com.volasoftware.tinder.dtos.RatingResponseDto;
+
+public interface RatingService {
+
+    //TODO
```

Comment:
Its best to remove comments from the code if they are resolved or don't serve any meaning to the other devs.